project-base/storefront/components/Basic/UserText/GrapesJsParser.tsx

```diff
@@ -58,7 +60,7 @@ export const GrapesJsParser: FC<GrapesJsParserProps> = memo(({ text, uuid }) =>
         return <UserText key={uuid} htmlContent={part} isGrapesJs />;
     };
 
-    return renderGrapesJsParts(text);
+    return <>{dividedParts.map(renderGrapesJsParts)}</>;
```

Comment:
I am not saying this is wrong, but I just want to know what motivated you for this change? (I dont see an error description in the commit message)

Reply:
It's not about fixing error. It's actual fix for the bug which I've discovered during fixing errors in console. You can see that any article is missing its body. That's because parser is broken. For example https://ssfwcc.prod.shopsys.cloud/samsung-galaxy-z-fold4-vyssi-vykon-a-lepsi-fotoaparaty-v-akci. Now it's fixed.